tensornetwork/backends/jax/jitted_functions_test.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import jax
+import pytest
+#pylint: disable=line-too-long
+from tensornetwork.backends.jax.jitted_functions import _generate_arnoldi_factorization
+jax.config.update('jax_enable_x64', True)
+
+
+def test_arnoldi_factorization():
+  np.random.seed(10)
+  D = 20
+  mat = np.random.rand(D, D)
+  x = np.random.rand(D)
+  dtype = np.float64
```

Comment:
Is it sufficient to just test the one dtype?

Reply:
no, good catch!